### PR TITLE
feat(react-native): add translations to SDK and DF app

### PR DESCRIPTION
**Pull request by khushal87**
- Added translations for the DF app and SDK texts.
- Fixed the error of translations on app start.

### CODE REVIEW

**Comment by santhoshvai** [comment]
What is the point of having de and es files? if its all a todo

**Comment by khushal87** [comment]
> What is the point of having de and es files? if its all a todo

I didn't add them, it was already there. :)

File: packages/react-native-sdk/src/components/call/LobbyView.tsx [hunk 1]
```diff
@@ -129,16 +133,22 @@ export const LobbyView = ({ joinCallButton }: LobbyViewType) => {
         )}
         <View style={styles.info}>
           <Text style={styles.infoText}>
-            You are about to join a call with id {call?.id} at Stream.{' '}
-            {participantsCount
-              ? `${participantsCount} participant(s) are in the call.`
-              : 'You are first to Join the call.'}
+            {t('You are about to join a call with id {{ callId }} at Stream.', {
+              callId: call?.id,
+            }) +
```
Comment (santhoshvai) [inline packages/react-native-sdk/src/components/call/LobbyView.tsx:138]: problematic to say at Stream in SDK actually :( 

I actually don't know what to do here expect saying lets pass a prop for info

at stream basically means DF app

Comment (santhoshvai) [reply packages/react-native-sdk/src/components/call/LobbyView.tsx:138]: I think the whole info text can be passed as prop 

Also we can add heading, subheading prop .. 

Expect for Join I don't see any other texts being useful in translations

File: sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx [hunk 2]
```diff
@@ -41,7 +42,7 @@ export const LobbyViewComponent = ({
           }}
         >
           <Text style={styles.anonymousButtonText}>
-            Join with your Stream Account
+            {t('Join with your Stream Account')}
```
Comment (santhoshvai) [inline sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx:45]: another problematic text

**Comment by khushal87** [comment]
> we must remove them.. especially from SDK.. any TODOs inside the SDK is not good after release :(

We didn't add any text on the SDK. Most of the texts are DF-specific. I can't think of any issue here.

File: sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx (hunk 2 above)
Comment (khushal87) [reply sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx:45]: This is DF app so its fine.

File: packages/react-native-sdk/src/components/call/LobbyView.tsx (hunk 1 above)
Comment (khushal87) [reply packages/react-native-sdk/src/components/call/LobbyView.tsx:138]: Good find. Let me fix this.

Comment (khushal87) [reply packages/react-native-sdk/src/components/call/LobbyView.tsx]: About the prop thing, what I believe is probably in the future is to design and keep LobbyView in the app itself and SDK won't export it in my opinion so it's fine for now.

File: sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx (hunk 2 above)
Comment (santhoshvai) [reply sample-apps/react-native/dogfood/src/components/LobbyViewComponent.tsx:45]: ah ok got you

**Comment by santhoshvai** [comment]
@khushal87 I got you sorry for the warn. 

But lets remove the DE and ES translations. Especially from SDK, exporting TODO texts is bad.

**Comment by MartinCupela** [comment]
> @khushal87 I got you sorry for the warn.
> 
> But lets remove the DE and ES translations. Especially from SDK, exporting TODO texts is bad.

We should probably move them to the dogfood apps?

**Comment by khushal87** [comment]
> > @khushal87 I got you sorry for the warn.
> > But lets remove the DE and ES translations. Especially from SDK, exporting TODO texts is bad.
> 
> We should probably move them to the dogfood apps?

Most of them are already in the DF app

**Pull request merged**